torchgeo/datasets/eurosat.py

```diff
@@ -74,16 +74,16 @@ class EuroSAT(NonGeoClassificationDataset):
         "test": "7ae5ab94471417b6e315763121e67c5f",
     }
     classes = [
```

Comment:
I think we should just remove this variable since the base class sets it automatically, but curious what @calebrob6 thinks

Reply:
done